app/enterprise/2.1.x/deployment/kong-for-kubernetes-enterprise.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Kong for Kubernetes Enterprise
+redirect_from: "/enterprise/2.1.x/kong-for-kubernetes/"
+---
+
+Kong provides two main Enterprise packages that you can deploy with Kubernetes,
+depending on your use case:
+
+* **Kong for Kubernetes Enterprise**: The Kong Ingress Controller with
```

Comment:
These product package names are confusing as we know. Maybe even changing "for" to "on" would slightly help?

Reply:
I agree with you 100%, but unfortunately that's not our call to make - even changing the "for" to "on" would create more confusion right now. Shane is already working on making this clearer, so until then, this is what we have to go with.

Leaving as is.